File: src/test/java/year_2023/october/easy/ToLowerCaseTest.java
package year_2023.october.easy;

import org.example.year_2023.october.easy.ToLowerCase;
import org.junit.Assert;
import org.junit.Test;

public class ToLowerCaseTest {
    @Test
    public void test() {
        test("hello", "Hello");
        test("here", "here");
        test("lovely", "LOVELY");
    }

    private void test(String exp, String act) {
        Assert.assertEquals(exp, ToLowerCase.toLowerCase(act));
    }
}

File: leetcode-python/src/test/easy/april/to_lower_case_test.py
import pytest

from src.solution.easy.april.to_lower_case import to_lower_case


@pytest.mark.parametrize(
    "input, expected", [
        ("Hello", "hello"),
        ("here", "here"),
        ("LOVELY", "lovely"),
        ("al&phaBET", "al&phabet")
    ]
)
def test_to_lower_case(input, expected):
    assert to_lower_case(input) == expected


if __name__ == '__main__':
    pytest.main()
